src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateLTMReferenceData.sql

```diff
@@ -0,0 +1,140 @@
+﻿CREATE PROCEDURE [dbo].[PopulateLTMReferenceData]
+(
+   @RunId int
+)
+AS
+-- =======================================================
+-- Author:      Sarma Evani
+-- Create Date: 19/10/2021
+-- Description: Populate LTM Reference Data
+-- =======================================================
+
+		BEGIN TRY
+
+			SET NOCOUNT ON
+
+		 DECLARE @LogID int
+
+		/* Start Logging Execution */
+
+		  INSERT INTO Mgmt.Log_Execution_Results
+			  (
+				RunId
+			   ,StepNo
+			   ,StoredProcedureName
+			   ,StartDateTime
+			   ,Execution_Status
+			  )
+		  SELECT 
+				@RunId
+			   ,'Step-3'
+			   ,'PopulateLTMReferenceData'
+			   ,getdate()
+			   ,0
+
+		   SELECT @LogID=MAX(LogId) FROM Mgmt.Log_Execution_Results
+		   WHERE StoredProcedureName='PopulateLTMReferenceData'
+			 AND RunId=@RunID
+
+
+		IF @@TRANCOUNT=0
+		BEGIN
+		BEGIN TRANSACTION
+
+				/*Loading Pledge Sector data */				
+				DELETE FROM [ASData_PL].[LTM_PledgeSector]
+				INSERT INTO [ASData_PL].[LTM_PledgeSector]
+				(
+					PledgeID,
+					SectorValue,
+					SectorDescription
+				) 
+				SELECT p.Id As PledgeID,l.FieldValue,l.FieldDesc 
+				FROM [ASData_PL].[LTM_Pledge] p
+				join [dbo].[ReferenceData] l on p.Sectors & l.FieldValue > 0 and l.Category = 'LevyTransferMatching' and l.FieldName = 'Sector' 
+				order by 1,cast(l.FieldValue as int) asc
+
+
+				/*Loading Pledge Level data */				
+				DELETE FROM [ASData_PL].[LTM_PledgeLevel]
+				INSERT INTO [ASData_PL].[LTM_PledgeLevel]
+				(
+					PledgeID,
+					LevelValue,
+					LevelDescription
+				) 
+				SELECT p.Id As PledgeID,l.FieldValue,l.FieldDesc 
+				FROM [ASData_PL].[LTM_Pledge] p
+				join [dbo].[ReferenceData] l on p.Levels & l.FieldValue > 0 and l.Category = 'LevyTransferMatching' and l.FieldName = 'Level' 
+				order by 1,cast(l.FieldValue as int) asc
+
+				/*Loading Pledge JobRole data */
+				DELETE FROM [ASData_PL].[LTM_PledgeJobRole]
+				INSERT INTO [ASData_PL].[LTM_PledgeJobRole]
+				(
```

Comment:
Can we either call the tables as Lookup (Lkup) as a suffix or Prefix , or keep them under Mtd Schema.